microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -291,6 +305,10 @@ def _kdump_test(self, node: Node, log: Logger) -> None:
                 )
                 time.sleep(10)
         if not stdout:
+            serial_console = node.features[SerialConsole]
+            serial_console.check_exception(
```

Comment:
How about add patterns into kernel panic, and just check panic here?

Reply:
This is not the same as a kernel panic. When the issue happened, the disk has an exception. Then the file system check of root filesystem failed. The system entered into 'initramfs'.